permits/templatetags/permits_extras.py

```diff
@@ -126,6 +126,16 @@ def can_always_be_updated(context):
 
 @register.simple_tag(takes_context=True)
 def can_download_archive(context):
-    return (
-        context["record"].archivist.groups.all() & context["user"].groups.all()
-    ).exists()
+    return services.can_download_archive(context["user"], context["record"].archivist)
+
+
+@register.simple_tag(takes_context=True)
+def can_archive(context):
+    department = models.PermitDepartment.objects.filter(
+        group__in=context["user"].groups.all()
+    ).first()
+    return department is not None and (
```

Comment:
The logic is correct, but there is one case to cover : in the DB, the superuser doesn't have a department., but still must have the permissions to archive